README.md

```diff
@@ -174,6 +175,22 @@ for await (const file of unpack(carReader)) {
 }
 ```
 
+### `ipfs-car/unpackStream`
+
+Takes an AsyncIterable and yields files to be consumed.
+
+```js
+import fs from 'fs'
+import { unpackStream } from 'ipfs-car/unpack'
+
+const inStream = fs.createReadStream(`${process.cwd()}/output.car`)
+
+const files = []
+for await (const file of unpackStream(inStream)) {
+  // Iterate over files
+}
+```
+
```

Comment:
We should document unpack functions with the blockstore

Reply:
lovely docs added in https://github.com/web3-storage/ipfs-car/pull/46/commits/a00e004b698cc42828e51869952d671bf7f9074b